Common/TableProducer/PID/pidTPC.cxx

```diff
@@ -122,6 +159,9 @@ struct tpcPid {
     enableFlag("Tr", pidTr);
     enableFlag("He", pidHe);
     enableFlag("Al", pidAl);
+    if (doprocessMcTuneOnData) {
```

Comment:
I would do the opposite, if enabled turn on the process function as well.

Reply:
If I'm understanding you right - is it possible to do that? I didn't think the process switches let you change the workflow topology during the init() based on table requirements like that (we would also need to disable the "standard" one in this case) but maybe there's some trick with process switches I didn't know about

Reply:
yes, you can tune the process functions based on the needs. That saves quite some overhead.
Something like doprocessMcTuneOnData.value = true; or so, it's very convenient.